proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -110,6 +111,9 @@ message VersionLocalState {
         // truth for that is the value at the root of WorkerDeploymentLocalState.
         map<int32, DeploymentVersionData> task_queues = 1;
     }
+
+    google.protobuf.Duration drainage_refresh_interval = 12;
+    google.protobuf.Duration drainage_visibility_grace_period = 13;
```

Comment:
Please educate me, why we need to store these values in the state? can't we read the latest config in a `SideEffect` or something and then schedule the timer based on that, as opposed to sticking to potentially old config.

Reply:
I didn't know any other way to read the config besides doing it in the fx